andela/ah-the-unsullied-frontend
# #162949166 Setup circle CI test runner

#### What does this PR do?
Sets up the circle CI test runner
#### Description of Task to be completed?
- Add a .circleci/config.yml file to the root directory of the project
- Add the circle ci badge to the README
#### How should this be manually tested?
1. Navigate to this link: [https://circleci.com/gh/andela/workflows/ah-the-unsullied-frontend](https://circleci.com/gh/andela/workflows/ah-the-unsullied-frontend) and view the build history
2. Check out the badge on the README file of the repo
#### Any background context you want to provide?
Since the develop branch does not have the .circleci/config.yml file, the circle ci badge shows a build failing result. However, the current branch where the configuration has been done has a passing result, as can be shown below:

![image](https://user-images.githubusercontent.com/23090268/52347899-16c22900-2a34-11e9-98f1-dd4df8e7b55c.png)

#### What are the relevant pivotal tracker stories?
[#162949166](https://www.pivotaltracker.com/story/show/162949166)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/23090268/52348138-a5cf4100-2a34-11e9-9556-2483c921d9df.png)
![image](https://user-images.githubusercontent.com/23090268/52349371-7c63e480-2a37-11e9-81e3-482bc5a19ecb.png)
